public/index.html

```diff
@@ -64,45 +70,48 @@
       .Pill {
         margin-left: 5px;
         font-weight: 500;
-        font-family: monospace !important;
         margin-bottom: 5px;
       }
 
-      .Pill span {
-        font-weight: 700;
-        font-size: 1rem;
-        padding-left: 6px;
-        box-sizing: content-box;
-      }
-
       .Pill span.material-icons {
-        font-size: 1.1rem;
+        font-size: 1.2rem;
       }
 
       .rot90 {
         transform: rotate(90deg);
         position: relative;
-        top: -2px;
+        top: -3px;
+      }
+
+      .ContainerStationPills :not(.BlankChip) .MuiChip-label {
+        font-family: monospace;
+        font-weight: 700;
+        font-size: 1.4rem;
+      }
+
+      .MuiChip-root .Badge.MuiChip-avatar {
```

Comment:
Sorry about that, my PR didn't change the margin or default height/width of the circle but I thought I fixed the small font. Anyway, it looks good now on my local.

Reply:
fwiw I don't think any of that was your doing. I think it was the version upgrade's fault.